### PR TITLE
Create webassembly pipeline 

### PR DESCRIPTION
**Pull request by Toudsour**
- Create webassembly pipeline

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[submodule "cmake/deps/emsdk"]
```
Comment (wenbingl) [inline .gitmodules:1]: I intentionally avoid the git submodule to decrease this project complexity.
Can we pass the emsdk DIR in the build command argument?

File: ci_build/azure-pipelines/mshost.yaml [hunk 2]
```diff
@@ -3,227 +3,269 @@
 #######
 
 jobs:
-- job: Linux
```
Comment (wenbingl) [inline ci_build/azure-pipelines/mshost.yaml:6]: why is every line changed? it's hard to review

File: .gitmodules (hunk 1 above)
Comment (Toudsour) [reply .gitmodules:1]: Sure, we can download emsdk in the pipeline. BTW, I am curious why git submodule will increase complexity.

File: ci_build/azure-pipelines/mshost.yaml (hunk 2 above)
Comment (Toudsour) [reply ci_build/azure-pipelines/mshost.yaml:6]: I format this file, only added lines: 230-271

File: CMakeLists.txt [hunk 3]
```diff
@@ -15,15 +15,13 @@ include(CheckLanguage)
...
 option(OCOS_ENABLE_PYTHON "Enable Python component building" OFF)
+option(OCOS_ENABLE_CTEST "Enable C++ test" ON)
 option(OCOS_ENABLE_TF_STRING "Enable String Operator Set" ON)
 option(OCOS_ENABLE_GPT2_TOKENIZER "Enable the GPT2 tokenizer building" ON)
 option(OCOS_ENABLE_SPM_TOKENIZER "Enable the SentencePiece tokenizer building" ON)
 option(OCOS_ENABLE_BERT_TOKENIZER "Enable the BertTokenizer building" ON)
 
-find_library(ONNXRUNTIME onnxruntime HINTS "${ONNXRUNTIME_LIB_DIR}")
-if ((NOT OCOS_ENABLE_PYTHON) AND (NOT ONNXRUNTIME))
-  message(FATAL_ERROR "Cannot find onnxruntime in the default library paths, please specify the ONNXRUNTIME_LIB_DIR.")
```
Comment (wenbingl) [inline CMakeLists.txt:25]: This is a pretty useful error message, is it replaced by some other message?

File: CMakeLists.txt [hunk 4]
```diff
@@ -98,6 +99,7 @@ endif()
 add_compile_options("$<$<C_COMPILER_ID:MSVC>:/utf-8>")
 add_compile_options("$<$<CXX_COMPILER_ID:MSVC>:/utf-8>")
 add_library(ortcustomops_static STATIC ${TARGET_SRC})
+target_include_directories(ortcustomops_static PUBLIC tokenizer)
```
Comment (wenbingl) [inline CMakeLists.txt:102]: so the tokenizer cannot be disabled in the compiling now?

File: CMakeLists.txt (hunk 3 above)
Comment (Toudsour) [reply CMakeLists.txt:25]: Actually, we check the onnxruntime library only when we build test, so I move them to line:226 after `OCOS_ENABLE_CTEST` condition

File: CMakeLists.txt (hunk 4 above)
Comment (Toudsour) [reply CMakeLists.txt:102]: Emm... We can discuss this more in the meeting. You can refer this question:[How to include all objects of an archive in a shared object?](https://stackoverflow.com/questions/2763988/how-to-include-all-objects-of-an-archive-in-a-shared-object?rq=1)

**Review by wenbingl — APPROVED** [review]

**Comment by Toudsour** [comment]
@wenbingl, there are some failed tests due to Eager model. After investigation, I can't figure out the root cause. (Maybe it's related to the new version of onnx -- 1.9)

**Pull request merged**